tests/Constraints/FactoryTest.php

```diff
@@ -76,42 +81,42 @@ public function constraintNameProvider()
      * @dataProvider invalidConstraintNameProvider
      *
      * @param string $constraintName
-     * @return void
      */
     public function testExceptionWhenCreateInstanceForInvalidConstraintName($constraintName)
     {
         $this->setExpectedException('JsonSchema\Exception\InvalidArgumentException');
         $this->factory->createInstanceFor($constraintName);
     }
 
-    public function invalidConstraintNameProvider() {
-      return array(
+    public function invalidConstraintNameProvider()
+    {
+        return array(
         array('invalidConstraintName'),
       );
     }
 
     /**
-     * @expectedException InvalidArgumentException
+     * @expectedException \InvalidArgumentException
```

Comment:
This is probably incorrect - `\JsonSchema\Exception\InvalidArgumentException` is a thing.